dist/filter-button/ds4/filter-button.css

```diff
@@ -0,0 +1,106 @@
+.filter-button-cell {
+  display: inline-block;
+  max-width: 258px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+button.filter-button,
+a.filter-link {
+  -webkit-box-align: center;
+          align-items: center;
+  border: none;
+  border-radius: 16px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  color: inherit;
+  display: -webkit-inline-box;
+  display: inline-flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+          flex-direction: column;
+  font-family: inherit;
+  font-size: 14px;
+  height: 32px;
+  -webkit-box-pack: center;
+          justify-content: center;
+  line-height: 1.2em;
+  margin: 0;
+  max-width: 280px;
+  min-width: 56px;
+  padding: 8px 12px;
+  text-align: center;
+  text-decoration: none;
+  vertical-align: bottom;
+  background-color: #eee;
+}
+button.filter-button:focus,
+a.filter-link:focus,
+button.filter-button:hover,
+a.filter-link:hover,
+button.filter-button:active,
+a.filter-link:active {
+  background-color: #e1e1e1;
+}
+.filter-button__cell,
+.filter-link__cell {
+  display: inline-block;
+  max-width: 258px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+a.filter-link:visited {
+  color: inherit;
+  color: #333;
+}
+button.filter-button--selected,
```

Comment:
Is there a reason to use `filter-button--selected` instead of `filter-button[aria-pressed=true]`? I know the modifier makes things a little easier in terms of a consistent specificity between the link and button versions, but we've gone with the enforcement of ARIA attributes up till now, so we should probably continue...?

Reply:
This was a straight copy from the way we had implemented pills. I'll update.

Reply:
Aha. That one must have slipped past first time around. Thanks.